extensions/hypersonic/src/features/todo-list/components/general-actions.tsx

```diff
@@ -0,0 +1,34 @@
+import { OpenPreferencesAction } from '@/components/open-preferences-action'
+import { Action, ActionPanel, Icon, openCommandPreferences } from '@raycast/api'
+import { OpenNotionAction } from './open-notion-action'
+import { ReauthorizeAction } from './reauthorize-action'
+import { WhatHaveIDoneAction } from './what-have-i-done-action'
+
+export function GeneralActions({
+  mutatePreferences,
+  notionDbUrl,
+}: {
+  mutatePreferences: () => void
+  notionDbUrl: string
+}) {
+  return (
+    <ActionPanel.Section>
+      <WhatHaveIDoneAction />
+      <OpenNotionAction notionDbUrl={notionDbUrl} />
+      <ReauthorizeAction />
+      <OpenPreferencesAction revalidate={mutatePreferences} />
+      <Action
+        title="Open Extension Preferences"
+        icon={Icon.Gear}
+        onAction={openCommandPreferences}
+        shortcut={{ modifiers: ['cmd'], key: ',' }}
+      />
+      <Action.CopyToClipboard
+        title="Drop us a Line"
+        icon={Icon.Heart}
+        content={'hypersonic@reboot.studio'}
+        shortcut={{ modifiers: ['cmd', 'shift'], key: 'm' }}
```

Comment:
I'm confused by this action. I was expecting it to open a webpage where I could reach out to you, but instead it copies the email address. Im not sure it's very intuitive. 

How about open your Twitter?

Reply:
Solved https://github.com/raycast/extensions/pull/3644/commits/a371988c42c5066c8f78cf0e6bd896346e1b950e